screens/CartScreen.js

```diff
@@ -0,0 +1,131 @@
+import { View, Text, StyleSheet, Pressable, ScrollView } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import { CartContext } from '../store/context';
+import { useContextSelector } from 'use-context-selector';
+
+function CartScreen(){
+  const cartItems = useContextSelector(CartContext, v => v.cartItems);
+  const getCartTotal = useContextSelector(CartContext, v => v.getCartTotal);
+  const removeFromCart = useContextSelector(CartContext, v => v.removeFromCart);
+
+  //decrease quantity of product by one in cart
+  function removeProduct(product){
+    removeFromCart(product);
+  }
+
+  return (
+    <ScrollView showsVerticalScrollIndicator = {false} bounces={false}>
+     <View style = {styles.root}>
+      
+      {cartItems.length === 0 && <Text style = {{textAlign:'center'}}>Cart is Empty</Text>}
+      {cartItems.map((product) => {
```

Comment:
When dealing with an unpredictable number of long lists, rendering directly via map is not recommended. Ensuring performance is crucial in React Native, so consider switching to using FlatList or SectionList for rendering long lists.